java/client/src/org/openqa/selenium/Platform.java

```diff
@@ -72,6 +72,13 @@ public Platform family() {
     }
   },
 
+  WIN10("windows 10", "win10") {
```

Comment:
it is good for merging at this point, but i'm awaiting others.  it's likely that we won't merge this until it actually comes out officially.